doc/user/content/get-started.md

```diff
@@ -5,216 +5,271 @@ menu: "main"
 weight: 3
 ---
 
-To help you get started with Materialize, we'll:
+This guide walks you through getting started with Materialize, from installing it to creating your first materialized view on top of streaming data. We'll cover:
 
-- Install, run, and connect to Materialize
-- Explore its API
-- Set up a real-time stream to perform aggregations on Wikipedia user edits.
+* Installing, running and connecting to Materialize
+
+* Connecting to a streaming data source
+
+* Creating a materialized view
+
+* Exploring common patterns like joins and time-windowing
+
+<p>
 
 {{< cta href="/katacoda/?intro-wikipedia" target="_blank" >}}
 Try the demo in your browser →
 {{</ cta >}}
 
-## Prerequisites
+## Install, run, connect
 
-To complete this demo, you need:
+Select an environment and follow the instructions to get the latest stable release of Materialize ({{< version >}}).
 
-- Command line and network access.
-- A [Materialize-compatible CLI](/connect/cli/). If you have Materialize installed `psql` works.
+{{< tabs >}}
+{{< tab "Docker">}}
 
-We also highly recommend checking out [What is Materialize?](/overview/what-is-materialize)
+1. Open a terminal and spin up a container running [`materialized`](https://hub.docker.com/r/materialize/materialized):
 
-## Install, run, connect
+    ```shell
+    docker run -p 6875:6875 materialize/materialized:{{< version >}} --workers 1
+    ```
 
-1. Install the `materialized` binary using [these instructions](/install).
+    This starts a process using one [worker thread](/connect/cli//#worker-threads) and listening on port 6875 by default.
 
-1. Run the `materialized` binary. For example, if you installed it in your `$PATH`:
+1. You can then connect to the running instance using any [Materialize-compatible CLI](/connect/cli/), like `psql` or `mzcli`. If you already have `psql` installed on your machine, connect using:
 
     ```shell
-    materialized -w 1
+    psql -U materialize -h localhost -p 6875 materialize
     ```
 
-    This starts the daemon listening on port 6875 using 1 worker.
+    Otherwise, you can find the steps to install and use your CLI of choice under [Install](/install/#cli-connections).
 
-1. Connect to `materialized` through your Materialize CLI, e.g.:
+{{< /tab >}}
+{{< tab "macOS">}}
 
+1. Assuming you're using [Homebrew](https://brew.sh/), open a terminal and run:
+    
     ```shell
-    psql -U materialize -h localhost -p 6875 materialize
+    brew install MaterializeInc/materialize/materialized
     ```
+    
+    **Note:** For a _curl_-based alternative, see [Install](/install/#curl).
```

Comment:
"curl" should be code, not italics.

For all of the psql commands, can you specify that the user should open another terminal to run them? That may be too basic for our user base, lot of our new employees haven't realized this until told.